### ISSUE TITLE
GRANTHER'S LUCKY HAMMER quest

### ISSUE THREAD

**Issue by StaH84** [WarEmu/WarBugs#6696]
Issue:
- not working potion as a reward (couldn't get the name)

**Comment by Ryanfawkez** [comment]
Intitiate's Screening Potion. Think that's either an absorb or a resist buff potion.

**Comment by Sayce78** [comment]
Issue: I could not interact with Granther's Coffin after someone else walked up to it, presumably for the same quest.

Details: I clicked on Granther's Coffin and the loot window appeared with the hammer for the quest. I could not click accept and I noticed another character by the coffin. I canceled out of the loot window but when I clicked on it again, I could no longer interact with the coffin (nothing happened when I clicked on it). I killed a few spirits and was still not able to interact with the coffin. I logged to character select and logged back in and then I could finally interact with the coffin and accept the hammer for the quest.
